node/network/gossip-support/src/lib.rs

```diff
@@ -52,6 +52,12 @@ const LOG_TARGET: &str = "parachain::gossip-support";
 // since the last authority discovery resolution failure.
 const BACKOFF_DURATION: Duration = Duration::from_secs(5);
 
+/// Duration after which we consider low connectivity a problem.
+///
+/// Especially at startup low connectivity is expected (authority discovery cache needs to be
+/// populated). After one minute of low connectivity a warning seems to be justified.
+const STARTUP_PERIOD: Duration = Duration::from_secs(60);
```

Comment:
Authority discovery takes around 10 minutes and thus, we should also choose 10 minutes here at least.

Reply:
Fair point - thanks! (Seriously, 10 minutes ...?)

Reply:
yes https://github.com/paritytech/substrate/blob/fc49802f263529160635471c8a17888846035f5d/client/authority-discovery/src/lib.rs#L88

Reply:
Yeah, found it already  :-)